sharding-core/sharding-core-common/src/main/java/org/apache/shardingsphere/core/strategy/algorithm/sharding/range/CustomRangeShardingAlgorithm.java

```diff
@@ -51,61 +49,39 @@
  * The sharding values will be divided into different partition by its value.
  * </p>
  */
-public final class RangeShardingAlgorithm implements StandardShardingAlgorithm<Long> {
+public final class CustomRangeShardingAlgorithm extends AbstractRangeShardingAlgorithm {
 
     private static final String PARTITION_RANGES = "partition.ranges";
 
     private Map<Integer, Range<Long>> partitionRangeMap;
```

Comment:
Hi, do you think it is appropriate to move `partitionRangeMap` to `AbstractRangeShardingAlgorithm `?

Reply:
> Hi, do you think it is appropriate to move `partitionRangeMap` to `AbstractRangeShardingAlgorithm `?
@tristaZero Before I submit the pr, I hava already tried to put the partitionRangeMap in the AbstractRangeShardingAlgorithm class, decorated with a protected identifier, but the CheckStyle detection prompt that it should be defined as a private type, and accessed through the getter/setter method. I think it is not elegant if we use the getter/setter method in the subclass to operate the partitionRangeMap.

Reply:
Got your idea, it is not a big deal, and thanks for the clarification. :)